pkg/util/provider/app/app.go

```diff
@@ -306,7 +306,7 @@ func getAvailableResources(clientBuilder coreclientbuilder.ClientBuilder) (map[s
 		}
 
 		healthStatus := 0
-		resp := client.Discovery().RESTClient().Get().AbsPath("/healthz").Do().StatusCode(&healthStatus)
+		resp := client.Discovery().RESTClient().Get().AbsPath("/healthz").Do(context.TODO()).StatusCode(&healthStatus)
```

Comment:
Shouldn't this be `context.Background()` or context passed as an argument to the function?

Reply:
You're right, Amshu. This function is referred within `StartControllers`. So, the context need not be passed as argument as there is no propagated usage. 

This should have been `context.Background()`. I will make the change.